docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+services:
+  app:
```

Comment:
Lets rename this `go-v8` so we can run commands against the container name instead of the ID

Reply:
I added a "container_name" property to the container in the docker-compose file.